app/gui/.env

```diff
@@ -0,0 +1,23 @@
+# CAUTION: this file is intentionally NOT ignored. If you want to override those variables, create an `.env.local` file.
+ENSO_IDE_AG_GRID_LICENSE_KEY=
```

Comment:
That's gonna break a lot of local dev setups and should definitely be documented/announced appropriately

Reply:
The point of this is to be a safe fallback that you don't have to worry about.